src/steamship/invocable/lambda_handler.py

```diff
@@ -218,15 +220,19 @@ def handler(internal_handler, event: Dict, _: Dict = None) -> dict:  # noqa: C90
 
 
 def create_handler(invocable_cls: Type[Invocable]):
-    """Legacy wrapper function for a Steamship invocable within an AWS Lambda function. Called by code within a
+    """Deprecated wrapper function for a Steamship invocable within an AWS Lambda function. Called by code within a
```

Comment:
Ah, so we'll want to update our templates

Reply:
Yes.  I was able to make it so that if you don't update the code, things should still work, but we should do a pass to remove it from templates (one-line delete)